doc/examples/segmentation/plot_chanvese.py

```diff
@@ -0,0 +1,25 @@
+""" This is an example to illustrate the us of the Chan-Vese Algorithm.
+"""
+
+from skimage.data import camera
+from skimage.segmentation.chanvese import chan_vese
+import numpy as np
+import matplotlib.pyplot as plt
+
+def _cv_main():
+    U0 = camera().astype('float')/255.
+    print np.max(U0)
```

Comment:
For simple uses of print like the one above, it's not even needed to import from ``__future__``, just using ``print(%)`` is enough